packages/repository-json-schema/src/__tests__/integration/build-schema.integration.ts

```diff
@@ -179,6 +180,23 @@ describe('build-schema', () => {
         expectValidJsonSchema(jsonSchema);
       });
 
+      it('properly converts any properties', () => {
+        @model()
+        class TestModel {
+          @property({
+            type: 'any',
+          })
+          anyProp: AnyType;
```

Comment:
The TypeScript typing should not be `AnyType`. Use `any` or `unknown` instead.

Reply:
good catch, updated to `any`

Reply:
Not yet. I'm still seeing `AnyType`.